app/views/shell/_top.html.erb

```diff
@@ -61,7 +61,6 @@
               --accent-brand: <%= SiteConfig.primary_brand_color_hex %>;
               --accent-brand-darker: <%= HexComparer.new([SiteConfig.primary_brand_color_hex]).brightness(0.85) %>;
               --accent-brand-lighter: <%= HexComparer.new([SiteConfig.primary_brand_color_hex]).brightness(1.1) %>;
-              --accent-brand-a10: #{rgba(<%= SiteConfig.primary_brand_color_hex %>, 0.1)};
```

Comment:
I got rid of this line since it was a bug - we can't use SCSS syntax and functions in HTML - this is something to figure out for later, @benhalpern